json/azurestorage_checklist.en.json

```diff
@@ -0,0 +1,120 @@
+{
+  "items": [
+    {
+      "category": "Redundancy",
+      "subcategory": "Platform Version",
+      "text": "Leverage a storagev2 account type for better performance and reliability",
+      "waf": "Reliability",
+      "guid": "cb8eb8c0-aa62-4a25-a495-6eaa8dc4a243",
+      "id": "49.1.1",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/storage/common/storage-account-upgrade?tabs=azure-portal"
+    },
+    {
+      "category": "Redundancy",
+      "subcategory": "Availablity",
+      "text": "Leverage GRS, ZRS or GZRS storage for the highest availability",
+      "waf": "Reliability",
+      "guid": "e05bbe20-9d49-4fda-9777-8424d116785c",
+      "id": "49.1.2",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/storage/common/storage-redundancy"
+    },
+    {
+      "category": "Redundancy",
+      "subcategory": "Failover",
+      "text": "Customer-Managed Failover",
+      "waf": "Reliability",
+      "guid": "2fa56c56-ad48-4408-be72-734c486ba280",
+      "id": "49.1.3",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/storage/common/storage-disaster-recovery-guidance"
+    },
+    {
+      "category": "Redundancy",
+      "subcategory": "Failover",
+      "text": "Microsoft-Managed Failover",
+      "waf": "Reliability",
+      "guid": "dc0590cf-65de-48e1-909c-cbd579266bcc",
+      "id": "49.1.4",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/storage/common/storage-disaster-recovery-guidance#microsoft-managed-failover"
+    },
+    {
+      "category": "Retention",
+      "subcategory": "Data Protection",
+      "text": "Enable Soft Delete",
+      "waf": "Reliability",
+      "guid": "a274faa1-abfe-49d5-9d04-c3c4919cb1b3",
+      "id": "49.1.5",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/storage/blobs/soft-delete-blob-enable?tabs=azure-portal"
+    }
+  ],
+  "categories": [

```

Comment:
@roshair It seems these sections were not available in the reference Json uploaded by Umar. Can you validate once and update?

Reply:
@infosatheesh2020 : I have removed the additional sections from both files, can you re-check once again and let me know if it is fine?